CHANGELOG.md

```diff
@@ -1,7 +1,32 @@
 # Changelog
 
-## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.15.1...HEAD)
-* Removed support for Airflow 1.10 [`#1128`](https://github.com/OpenLineage/OpenLineage/pull/1128) [@mobuchowski](https://github.com/mobuchowski)
+## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.16.0...HEAD)
+## [0.16.0](https://github.com/OpenLineage/OpenLineage/compare/0.15.1...0.16.0) - 2022-11-1
+### Added
+* Airflow: add `dag_run` information to Airflow version run facet [`$1133`](https://github.com/OpenLineage/OpenLineage/pull/1133) [@fm100](https://github.com/fm100)  
+    *Adds the Airflow DAG run ID to the `taskInfo` facet, making this additional information available to the integration.*
+* Airflow: add `LoggingMixin` to extractors [`#1149`](https://github.com/OpenLineage/OpenLineage/pull/1149) [@JDarDagran](https://github.com/JDarDagran)  
+    *Adds a `LoggingMixin` class to the custom extractor to make the output consistent with general Airflow and OpenLineage logging settings.*
+* Airflow: add default extractor [`#1162`](https://github.com/OpenLineage/OpenLineage/pull/1162) [@mobuchowski](https://github.com/mobuchowski)  
+    *Adds a `DefaultExtractor` to support the default implementation of OpenLineage for external operators without the need for custom extractors.*
+* Airflow: add `on_complete` argument in `DefaultExtractor` [`#1188`](https://github.com/OpenLineage/OpenLineage/pull/1188) [@JDarDagran](https://github.com/JDarDagran)  
+    *Adds support for running another method on `extract_on_complete`.*
+* SQL: reorganize the library into multiple packages [`#1167`](https://github.com/OpenLineage/OpenLineage/pull/1167) [@StarostaGit](https://github.com/StarostaGit) [@mobuchowski](https://github.com/mobuchowski)   
+    *Splits the SQL library into a Rust implementation and foreign language bindings, easing the process of adding language interfaces. Also contains CI fix.*
+
+### Changed
+* Airflow: remove support for Airflow 1.10 [`#1128`](https://github.com/OpenLineage/OpenLineage/pull/1128) [@mobuchowski](https://github.com/mobuchowski)  
```

Comment:
You'll want to move any remove changes under the section `Removed`, see [How do I make a good changelog?](https://keepachangelog.com/en/1.0.0/#how) (ex: Marquez [`CHANGELOG.md`](https://github.com/MarquezProject/marquez/blob/main/CHANGELOG.md#0210---2022-03-03))

Reply:
Thanks for catching these issues, @wslulciuc . RFR again when you have time.